brave-jdbc/src/test/java/com/github/kristofa/brave/jdbc/BraveJdbcBenchmarks.java

```diff
@@ -0,0 +1,115 @@
+package com.github.kristofa.brave.jdbc;
+
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OperationsPerInvocation;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Threads;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+import java.sql.ResultSet;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Benchmarks for testing the overall impact of using the p6 driver spy with Zipkin tracing enabled vs. the raw JDBC
+ * driver.
+ */
+@Measurement(iterations = 10, time = 1)
+@Warmup(iterations = 10, time = 1)
+@Fork(1)
+@BenchmarkMode(Mode.Throughput)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@Threads(1)
+@State(Scope.Benchmark)
+public class BraveJdbcBenchmarks {
```

Comment:
This depends on a bunch of other classes in the test project, so not sure it's worth it to try to bundle those up as shareable so that this class could be in the benchmarks module.